app/components/trial_role_warning_component/view.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if @current_user.trial?  %>
+  <%= govuk_notification_banner(title_text: t("trial_role_warning.title")) do |banner| %>
+    <% banner.with_heading(text: t("trial_role_warning.heading")) %>
+    <%= t("trial_role_warning.content_html", link_url: '/404') %>
```

Comment:
I think this is all we'll need to change once https://github.com/alphagov/forms-admin/pull/646 is merged
```suggestion
    <%= t("trial_role_warning.content_html", link_url: user_upgrade_request_path) %>
```